__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -221,6 +276,134 @@ export const WithInitialExpandedIndex: StoryComponentType = {
     },
 };
 
+/**
+ * An Accordion can be animated using the `animated` prop. This
+ * animation includes the caret, the expansion/collapse, and the last
+ * section's border radius. In this example, animated accordions with
+ * different corner kinds are shown to demonstrate the border radius transition,
+ * as well as accordions with \`allowMultipleExpanded\` set to \`false\`, and
+ * an accordion with sections of different heights.
+ *
+ * If the user has `prefers-reduced-motion` opted in, this animation should
+ * be disabled. This can be done by passing `animated={false}` to
+ * the Accordion.
+ *
+ * If `animated` is specified both here in the Accordion
+ * and within a child AccordionSection component, the Accordion's
+ * `animated` value is prioritized.
+ */
+export const WithAnimation: StoryComponentType = {
```

Comment:
**suggestion:** Disable this chromatic snapshot as we cannot visually test this in chromatic.